src/fs.ts

```diff
@@ -48,21 +48,17 @@ export async function writeJson(
   data: object,
   cwd = '.',
 ): Promise<void> {
-  try {
-    const text = JSON.stringify(data, null, '  ');
-    return writeText(path, text, cwd);
-  } catch (e) {
-    return;
-  }
+  const text = JSON.stringify(data, null, '  ');
+  return await writeText(path, text, cwd);
```

Comment:
It wasn't possible to hit this `catch` block a `writeText` already smothers any errors from writing the file.